crates/lang/tests/ui/contract/fail/constructor-payable-invalid-1.rs

```diff
@@ -6,7 +6,7 @@ mod contract {
     pub struct Contract {}
 
     impl Contract {
-        #[ink(constructor, payable)]
+        #[ink(constructor, payable = true)]
```

Comment:
Why change the existing test instead of creating a new one for this case?

Reply:
This test was added as a new one [in this commit](https://github.com/paritytech/ink/pull/1065/commits/fb4f13663de38b1b71b64bc2db13dbdf3f1d597c). There was a `git mv` in this [earlier commit](https://github.com/paritytech/ink/pull/1065/commits/2acf3c56896ab193a730ab88898587703bb683f4) though, maybe that's causing the confusion?

Either way, the [history](https://github.com/paritytech/ink/commits/hc-make-constructors-non-payable/crates/lang/tests/ui/contract/fail/constructor-payable-invalid-1.rs)  for this file shows it was only introduced in this PR, so it should be fine imo.